manifest.json

```diff
@@ -17,27 +12,31 @@
   },
   "content_scripts":          [
     {
-      "all_frames": true,
       "js":         [
         "env.js",
         "resources/scripts/console.js",
         "view/general.js",
         "view/showLocator.js",
         "inspection/contentScript.js"
       ],
-      "matches":    ["\u003Call_urls>"]
+      "matches": ["\u003Call_urls>"]
     }
   ],
-  "description":              "Kick your web automation into high gear with Nexial WebEZ!",
+  "description":"Kick your web automation into high gear with Nexial WebEZ!",
   "icons":                    {
     "16":  "resources/img/nexial-16.png",
     "32":  "resources/img/nexial-32.png",
     "64":  "resources/img/nexial-64.png",
     "128": "resources/img/nexial-128.png"
   },
-  "manifest_version":         2,
+  "manifest_version":         3,
   "name":                     "Nexial WebEZ",
-  "version":                  "0.9.1",
-  "permissions":              ["activeTab", "contextMenus"],
-  "web_accessible_resources": ["resources/style/nexial.css", "view/dialogInjector.js"]
+  "version":                  "1.0.0",
+  "permissions": ["activeTab","contextMenus","storage"],
```

Comment:
is "storage" permission needed?

Reply:
Storage permission is required to access chrome.storage